docs/source/tutorials_source/platform/tutorial_cropped_objects_metadata.py

```diff
@@ -0,0 +1,187 @@
+"""
+
+.. _lightly-tutorial-cropped-objects-metadata
+
+Tutorial 6: Find false negatives of object detection
+=============================================
+
+    In object detection applications, it can happen that the detector does not detect an object
+    because it did not see any examples of this or similar objects yet.
+    This is especially a problem in warehouse and retail applications,
+    as new products get added to the shelves every day. These new products with new appearence
+    often have a low so-called objectness score. Currently, finding these objects falsely detected
+    as non-objects needs a lot of manual labour, especially when having 1000s of new images coming
+    in every day. Lightly can make this work easier in a two-step approach:
+
+    1. A human finds one false negative and adds it to the missing examples.
+    2. Lightly finds all similar images which are also false negatives.
+    Thus it can propose to directly also add them as missing examples.
+
+    If there are e.g. 9 similar images for each missing example found by a human,
+    Lightly can speed up the process by a factor of 10.
+
+What you will learn
+--------------------
+
+* How to use an object detection model to crop objects out of a full image and save them as images.
+* How to save the object detection scores as metadata in a Lightly format.
+* How to upload the cropped images with their corresponding metadata to the webapp.
+* How to use the webapp to find false negatives and similar examples easily.
+
+Requirements
+-------------
+
+You can use your own dataset or the one we provide for this tutorial. The dataset
+we will use is the `SKU110k dataset <https://github.com/eg4000/SKU110K_CVPR19>`_ showing store shelves.
+The tutorial is computationally expensive if you run it on the full 110k images of products,
+thus we recommend running it on a subset of the dataset. E.g. copy 100 images from one folder to
+a new folder and use the path to the latter folder as input. Alternatively, run it on your own dataset.
+
+For this tutorial the `lightly` pip package needs to be installed:
+
+.. code-block:: bash
+
+    # Install lightly as a pip package
+    pip install lightly
+
+Steps
+-------------
+
+The steps of this tutorial are quite straightforward:
+    1. Define the dataset in form of a LightlyDataset of torch tensors. You need to provide the path to the dataset images.
+    2. Define a pretrained object detection model. We use the retina net trained on COCO 2017. As it was not pretrained on a retail dataset, its performance is not state-of-the art. Nonetheless, it is sufficient for this tutorial and very easy to use.
+    3. Predict with the model on the dataset.
```

Comment:
Is this a correct sentence? Why not "Generate predictions for each image in the dataset"